themes/root/templates/RecordDriver/AbstractBase/export-marc.phtml

```diff
@@ -1 +1,7 @@
-<?=$this->driver->getMarcRecord()->toRaw()?>
+<?php
+$marc = $this->driver->getMarcReader()->toISO2709();
```

Comment:
I think you missed this `toISO2709()` call in your refactoring; might want to double check if there are any others.

Reply:
Now fixed. I think I got the others right.